Configuration/StandardSequences/python/Digi_cff.py

```diff
@@ -83,3 +83,24 @@ def _modifyEnableHcalHardcode( theProcess ):
 
 from Configuration.Eras.Modifier_hcalHardcodeConditions_cff import hcalHardcodeConditions
 modifyEnableHcalHardcode_ = hcalHardcodeConditions.makeProcessModifier( _modifyEnableHcalHardcode )
+
+
+#phase 2 ecal pedestals
+def _modifyEcalPedestals( process ):
+    process.load("SimCalorimetry.EcalSimProducers.esEcalLiteDTUPedestalsProducer_cfi")
+
+    from CondCore.CondDB.CondDB_cfi import CondDB
+    CondDB.connect = cms.string('sqlite_file:SimCalorimetry/EcalSimProducers/data/simPulseShapePhaseII.db')
```

Comment:
@christopheralanwest @ggovi
This is in direct and flagrant violation of https://github.com/cms-sw/cmssw/issues/27393

Reply:
Indeed. If this payload is needed, please provide a validation of the new tag and we can add it to the global tag. Can the new tag be used independent of this PR or must the code and conditions be changed simultaneously?

Reply:
Well, the code compiles and runs even without the tag, but the results are non physical because the shape of phase1 is much larger. Indeed we are trying to have this "development" branch as a temporary workflow, because the reco part must be finished. it would be nice to have the code shared with other people. 
The new pulse shape is used only in ecal_devel Era, in this, let's say, sub workflow which will be used by other ecal groups. Thus, what is the best? merge without the new pulse and test the workflow (i mean reading back the new digi collections) but with a wrong shape, or it is possible to put it somewhere temporary? if not we can ask for the ALCA approval. Thanks for the advices.

Reply:
The current Phase 2 GT `112X_mcRun4_realistic_v2` contains the tag `EcalSimPulseShapePhaseI`, the same GT that is used in the 2018 GT. Can we use the `EcalSimPulseShapeRcd` payload contained in `simPulseShapePhaseII.db` even without this PR or would that give incorrect results? I'm not interested in merging this PR prior to updating the DB payload but rather the reverse.

Reply:
OK we will upload the payload contained in simPulseShapePhaseII

Reply:
the pulse shape is now on the DB, thus, as you can see here https://github.com/dariosol/cmssw/blob/b9214d31f4c48eb4765fd6ceeb1f890fd9b05422/SimCalorimetry/Configuration/python/ecalDigiSequence_cff.py#L31
we take the pulse shape as a new tag of the record EcalSimPulseShapeRcd. 
Hope this is better than before